Arrowgene.Ddon.GameServer/Handler/CraftRecipeGetGradeupRecipeHandler.cs

```diff
@@ -18,28 +18,24 @@ public CraftRecipeGetGradeupRecipeHandler(DdonGameServer server) : base(server)
 
         public override S2CCraftRecipeGetCraftGradeupRecipeRes Handle(GameClient client, C2SCraftRecipeGetCraftGradeupRecipeReq request)
         {
-            List<CDataMDataCraftGradeupRecipe> allRecipesInCategory = Server.AssetRepository.CraftingGradeUpRecipesAsset
+            var categoryRecipes = Server.AssetRepository.CraftingGradeUpRecipesAsset
```

Comment:
Should a TODO be added somewhere to further restrict the potential recipes beyond just by category? i.e. only those recipes that are related to items in the user's bag+storage?